src/main/java/com/shaft/gui/element/ElementActions.java

```diff
@@ -33,2118 +33,2132 @@
 import java.util.concurrent.TimeUnit;
 
 public class ElementActions {
```

Comment:
Please do not restyle the entire class file (just the lines you've added) as GitHub is detecting that it's all new.